src/urg_node.cpp

```diff
@@ -116,7 +116,7 @@ void UrgNode::initSetup()
 
   diagnostic_updater_.add("Hardware Status", this, &UrgNode::populateDiagnosticsStatus);
 
-  this->set_on_parameters_set_callback(std::bind(&UrgNode::param_change_callback, this,
+  this->add_on_set_parameters_callback(std::bind(&UrgNode::param_change_callback, this,
```

Comment:
I think we need to hold a reference to the returned shared pointer, otherwise the callback may be unregistered.